src/main/java/net/praqma/stash/plugins/tracey/components/impl/EiffelProtocolMessage.java

```diff
@@ -0,0 +1,130 @@
+package net.praqma.stash.plugins.tracey.components.impl;
+
+import com.atlassian.stash.repository.Repository;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.util.JsonFormat;
+import net.praqma.stash.plugins.tracey.exceptions.ProtocolServiceException;
+import net.praqma.tracey.protocol.eiffel.events.EiffelSourceChangeCreatedEventOuterClass;
+import net.praqma.tracey.protocol.eiffel.factories.EiffelSourceChangeCreatedEventFactory;
+import net.praqma.tracey.protocol.eiffel.models.Models;
+import net.praqma.utils.parsers.cmg.api.CommitMessageParser;
+import net.praqma.utils.parsers.cmg.impl.Jira;
+import org.osgi.framework.FrameworkUtil;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.util.Dictionary;
+
+import static org.eclipse.jetty.io.nio.SelectorManager.LOG;
```

Comment:
Please use default logger as it is done in other classes and remove jetty from pom.xml - as far as I can tell this is the only place you are using it so there is no good reason to bring jetty as dependency only to use log from there

private static final Logger LOG = LoggerFactory.getLogger(EiffelProtocolMessage.class);